src/radical/pilot/pytask.py

```diff
@@ -0,0 +1,77 @@
+
+__copyright__ = "Copyright 2013-2016, http://radical.rutgers.edu"
+__license__   = "MIT"
+
+import functools
+
+from .  import utils     as rpu
+
+TASK = dict 
+_SER = rpu.Serializer()
+
+
+class PythonTask(object):
```

Comment:
Why do we actually have a class `PythonTask` if that is only holding two methods which are not even class methods (don't take `self` as argument)?

Reply:
This class is for wrapping purposes, it acts as a function wrapper without the need to decorate. Example: 

```python
def AA(func:Callable, x= random.random()):
    eq = func(x)
    print (eq * 4)


def AB(z):
    import random
    return random.random() * z


def AC(y):
    import time
    return time.time()


wrapped_function = partial(AA, AB)

PythonTask(wrapped_function)
```

Also, we might need it to wrap any type of function with or without `args` and `kwargs` internally: for example `RP-Parsl` executor.